rakefile-dotnet.rb

```diff
@@ -258,6 +268,12 @@
   cmd.parameters "pack #{nuspec} -Version #{version}#{version_suffix} -OutputDirectory #{output}"
 end
 
+desc "create the nuget package with the .NET Standard"
+exec :pack => [:build, output] do |cmd|
+  cmd.command = nuget_command
+  cmd.parameters "pack #{dotnet_nuspec} -Version #{version}-dotnet#{version_suffix} -OutputDirectory #{output}"
+end
+

```

Comment:
I'm more than happy to switch to `netstd` as the moniker everywhere, but we seem to be delaying that and going with `dotnet` everywhere for now, which is why I suggested `_DOTNET`. I think we should do one or the other, but not mix.

The problem is, we'll need to be able to control the version suffix for the net40 package independently of the dotnet package. E.g. when we are pushing out an RC of the net40 package, I don't think we'll want to use RC in the version suffix of the net40+netstd package. And likewise, when we push out net40 RTM, we'll want the net40+netstd to still have a pre-release suffix. I don't think that we want `dotnet` in the version name there, but perhaps there's an idea there which I don't understand?